code/game/objects/items/stacks/wrap.dm

```diff
@@ -203,3 +211,10 @@
 	w_class = WEIGHT_CLASS_TINY
 	throw_speed = 3
 	throw_range = 5
+
+/obj/item/c_tube/attack(mob/living/target_mob, mob/living/user, params)
+	. = ..()
+	balloon_alert(user, "you bap them on the head")
```

Comment:
i think "bapped!" would make for a better (goofier?) balloon alert, but that's just me

Reply:
> i think "bapped!" would make for a better (goofier?) balloon alert, but that's just me

I think a bapped! would be better for whoever gets bapped and not the bapper

Reply:
I decided to remove it since there is a sound and visual message.